stripper/ze_dreamin_v3_3.cfg

```diff
@@ -1,3 +1,203 @@
+	"startvalue" "1"
+	"origin" "69 69 69"
+	"OnGetValue" "stage_case,InValue,,0,-1"
+}
+
+add:
+{
+	"classname" "point_template"
+	"targetname" "stage_system_template"
+	"Template01" "stage_counter"
+	"spawnflags" "2"
+	"origin" "69 69 69"
+}
+
+add:
+{
+	"classname" "info_target"
+	"targetname" "stage_counter_init"
+	"spawnflags" "0"
+	"origin" "69 69 69"
+	"OnUser1" "stage_system_template,ForceSpawn,,0,-1"
+	"OnUser1" "stage_counter,AddOutput,classname info_target,0.03,-1"
+	"OnUser1" "!self,Kill,,0.06,-1"
+}
+
+add:
+{
+	"classname" "logic_case"
+	"targetname" "stage_case"
+	"Case01" "1"
+	"Case02" "2"
+	"Case03" "3"
+	"case04" "4"
+	"origin" "69 69 69"
+	"OnCase01" "st1_trigger,Trigger,,0.2,1"
+	"OnCase02" "st2_trigger,Trigger,,0.2,1"
+	"OnCase03" "st3_trigger,Trigger,,0.2,1"
+	"OnCase04" "st4_trigger,Trigger,,0.2,1"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "stage_counter_init,FireUser1,,0,-1"
+		"OnMapSpawn" "stage_counter,GetValue,,0.02,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"hammerid" "805869"
+	}
+	delete:
+	{
+		"OnPressed" "brush_*Enable0-1"
+		"OnPressed" "brush_4Disable0.1-1"
+	}
+	insert:
+	{
+		"OnPressed" "stage_counter,SetValue,4,0,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"hammerid" "805872"
+	}
+	delete:
+	{
+		"OnPressed" "brush_*Enable0-1"
+		"OnPressed" "brush_3Disable0.1-1"
+	}
+	insert:
+	{
+		"OnPressed" "stage_counter,SetValue,3,0,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"hammerid" "805876"
+	}
+	delete:
+	{
+		"OnPressed" "brush_*Enable0-1"
+		"OnPressed" "brush_2Disable0.1-1"
+	}
+	insert:
+	{
+		"OnPressed" "stage_counter,SetValue,2,0,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"hammerid" "805880"
+	}
+	delete:
+	{
+		"OnPressed" "brush_*Enable0-1"
+		"OnPressed" "brush_1Disable0.1-1"
+	}
+	insert:
+	{
+		"OnPressed" "stage_counter,SetValue,1,0,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "st3_clear"
+	}
+	delete:
+	{
+		"OnStartTouch" "brush_*Enable0-1"
+		"OnStartTouch" "brush_4Disable0.01-1"
+	}
+	insert:
+	{
+		"OnStartTouch" "stage_counter,SetValue,4,0,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_once"
+		"hammerid" "381778"
+	}
+	delete:
+	{
+		"OnStartTouch" "brush_3Disable0.11"
+		"OnStartTouch" "brush_*Enable01"
+	}
+	insert:
+	{
+		"OnStartTouch" "stage_counter,SetValue,3,0,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "clear_trigger"
+	}
+	delete:
+	{
+		"OnStartTouch" "brush_2Disable0.1-1"
+		"OnStartTouch" "brush_*Enable0-1"
+	}
+	insert:
+	{
+		"OnStartTouch" "stage_counter,SetValue,2,0,-1"
+	}
+}
+
+// cleanup old physics level system
+filter:
+{
+	"classname" "func_brush"
+	"targetname" "/brush_.*/"
+}
+
+filter:
+{
+	"classname" "trigger_once"
+	"targetname" "/trigger_.*/"
+}
```

Comment:
This is too broad, it will catch `trigger_def_1` as well. Use `\d` to only match a digit.